src/dmd/dtemplate.d

```diff
@@ -6515,7 +6515,16 @@ extern (C++) class TemplateInstance : ScopeDsymbol
 
                     if (ea.op == TOK.variable)
                     {
-                        /* This test is to skip substituting a const var with
+                        /* If the parameter is a function that is not called
+                         * explicitly then takes it a dsymbol.
```

Comment:
```
then it takes a dsymbol
```

Reply:
I meant "as" I think

Reply:
This is still grammatically incorrect.

Reply:
you mean "takes" ?

Reply:
"then it takes", not "then takes it"

Reply:
before I force push again, is this alright ?

```
If the parameter is a function that is not called explicitly then take it as a dsymbol.
```

Reply:
Hmm, looking at the code in context a bit more, probably:

```
If the parameter is a function that is not called explicitly, i.e. `foo!func` as opposed to `foo!func()`, then it is a dsymbol, not the return value of `func()`
```
is best.